docs/sequence_diagrams/notification-ProcessesOverview.plantuml

```diff
@@ -0,0 +1,82 @@
+@startuml
+' declate title
+title 10.1. Notification Process Overview
+autonumber
+
+' declare actors
+boundary "Central Admin API" as CLAPI
+control "Central Event Processor" as CEP
+control "Notification Handler" as NOTIFIER
+collections "Notification-Topic" as TOPIC_NOTIFY
+database "CEP Store" as DB
+
+box "Central Services" #LightYellow
+participant CLAPI
+participant TOPIC_NOTIFY
+end box
+box "Central Event Processing Services" #LightGreen
+participant CEP
+participant DB
+end box
+box "Notifier Services" #LightBlue
+participant NOTIFIER
+end box
+
+' start flow
+group Notification Process
+    activate CEP
+    note right of TOPIC_NOTIFY #LightGrey
+        Events are published
+        to Notification Topic
+        by Central Services
+    end note
+    TOPIC_NOTIFY <- CEP: Consumes notification messages
+    activate TOPIC_NOTIFY
+    deactivate TOPIC_NOTIFY
+    CEP -> CEP
+    note right of CEP #LightGrey
+        Filter notification messages on event types
+    end note
+
+    alt Messages event type that qualify
+        CEP -> CLAPI: Request additional info for notification\n<color #FF0000><b>Error codes:</b> 2001, 2003 </color>
+        activate CLAPI
+        note left of CEP #LightGrey
+            API call to Central Admin API
+        end note
+        deactivate CLAPI
+        CEP -> DB: Persist message related data to CEP store
+        activate DB
+        deactivate DB
+        CEP -> CEP
+        note right of CEP #LightGrey
+            Invoke the Rule Engine.
+            Evaluate the facts.
+        end note
+
+        alt Qualifying messages will trigger a notification event
+            CEP -> CEP
+            note right of CEP #LightGrey
+                Notification messages is constructed.
+            end note
+            CEP -> DB: Persist new notification to CEP Store
+            activate DB
+            deactivate DB
+            TOPIC_NOTIFY <- CEP: Place message to Notification-Topic
+            activate TOPIC_NOTIFY
+        else Message does not qualify for notification event (Ignore)
+        end
+    else Message event type does not qualify for notification action (Ignore)
+    deactivate CEP
+    end
+
+    group Notification event
+    TOPIC_NOTIFY <-- NOTIFIER: Consumes messages from Notification Topic for processing.
+    note left of NOTIFIER #LightGrey
+        Notifier Handler to process topic and
```

Comment:
As this is not longer labeled Notifier Handler, please change to something like:
Notifier (e.g. email) to process topic and
send notifications/alerts

Reply:
Updated to Notifier on all diagrams.